packages/jest-preset-default/README.md

```diff
@@ -24,8 +24,6 @@ npm install @wordpress/jest-preset-default --save-dev
 
 #### Brief explanations of options included
 
-* `coverageDirectory` - the directory where Jest outputs its coverage files is set to `coverage`.
```

Comment:
We need to update changelog as this looks as a breaking change, or we can keep it as is 😄